### ISSUE TITLE
Binary for Armv8 (RPI 5)

### ISSUE THREAD

**Issue by p1r473** [sharkdp/fd#1551]
Hi, I tried the arm64 build and it didnt seem to work oin the RPI. I ended up building from source
Are you able to provide a binary though?

**Comment by tavianator** [comment]
What does "didn't work" mean exactly?

I'm guessing this is a dupe of #1085, which we intended to fix for 10.0 but will not be properly fixed until the next release (probably soon)

**Comment by p1r473** [comment]
Yes, sorry dupe of #1085
Same error
```
<jemalloc>: Unsupported system page size
<jemalloc>: Unsupported system page size
memory allocation of 5 bytes failed
[1]    28699 IOT instruction (core dumped)  ./fd-v8.4.0-aarch64-unknown-linux-gnu/fd
``
